Jamulus.entitlements

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+
+	<key>com.apple.security.cs.allow-dyld-environment-variables</key>
+	<true/>
+	<key>com.apple.security.cs.disable-library-validation</key>
+	<true/>
+	<key>com.apple.security.device.audio-input</key>
+	<true/>
+	<key>com.apple.security.files.user-selected.read-write</key>
+	<true/>
+	<key>com.apple.security.network.client</key>
```

Comment:
Would the server mode also need something else to allow it to listen on a port?

Reply:
Good catch - I'll update the server entitlements part.

Reply:
Could Jamulus be run in a sand box if it's installed outside of the app store? If that's possible, we might also consider doing that for security reasons - if that's not too much work.

Reply:
'Tis done :)